### PR TITLE
fix(Bastion): prevent disabling movement after getting out of sentry

### PR DESCRIPTION
**Pull request by arvrianmaybe**
sounds confusing a glitch appears when you enter sentry and get out of sentry way to quickly which causes a bug where you have no move speed at all. Its not huge but can be kinda annoying, (You can re-enter and go out to fix.)

### CODE REVIEW

**Comment by arvrianmaybe** [comment]
quick vid demonstrating the issue

https://github.com/Overwatch-1-Emulator/ow1-emulator/assets/106389239/70cb4703-b37b-44fc-b421-093430995eaf

**Comment by Ciach0** [comment]
This was a safeguard I've added to fix another bug, sometimes the bug would allow you to move in sentry form

File: src/heroes/bastion/sentry.opy [hunk 1]
```diff
@@ -16,8 +16,8 @@ rule "[bastion/sentry.opy]: Initialize sentry mode":
 
     eventPlayer.disallowButton(Button.SECONDARY_FIRE)
 
-    waitUntil(eventPlayer.isOnGround(), Math.INFINITY)
-    eventPlayer.setMoveSpeed(0)
+    #waitUntil(eventPlayer.isOnGround(), Math.INFINITY)
+    #eventPlayer.setMoveSpeed(0)
```
Comment (MaxwellJung) [inline src/heroes/bastion/sentry.opy:20]: Wouldn't the fix be to conditionally set movespeed to 0 only when they're in sentry mode?
```suggestion
    waitUntil(eventPlayer.isOnGround(), Math.INFINITY)
    if eventPlayer.isUsingAbility1():
        eventPlayer.setMoveSpeed(0)
```

**Review by MaxwellJung — APPROVED** [review]
Good find; thanks for contribution!

**Pull request merged**